assets/prototype/application/ui/components/forms/formModal/types.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FormProps, FormRenderProps } from 'react-final-form';
+import { FormApi } from 'final-form';
+
+export interface FormModalProps extends FormProps {
+	FormComponent: React.FC;
+	onClose: (e?: any) => void;
```

Comment:
can this use `React.SyntheticEvent` instead or will that cause issues?

Reply:
That's not guaranteed I noticed, that's why I set it to `any`